File: game-admin/src/main/java/com/dkm/seed/entities/bo/SeedEntity.java
package com.dkm.seed.entities.bo;

import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableName;
import java.math.BigDecimal;
import kotlin.Metadata;
import kotlin.jvm.internal.Intrinsics;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.jetbrains.annotations.NotNull;
import org.jetbrains.annotations.Nullable;

@AllArgsConstructor
@NoArgsConstructor
@Data
@TableName("tb_seed")
public final class SeedEntity {
   @TableId
   private long seedId;
   @Nullable
   private String seedName;
   @Nullable
   private Integer seedGrade;
   @Nullable
   private Integer seedProdgold;
   @Nullable
   private BigDecimal seedProdred;
   @Nullable
   private Integer seedExperience;
   @Nullable
   private Integer seedGold;
   @Nullable
   private String seedImg;


}

File: game-admin/src/main/java/com/dkm/seed/mapper/bo/SeedEntity.kt
package com.dkm.seed.mapper.bo

import com.baomidou.mybatisplus.annotation.TableId
import com.baomidou.mybatisplus.annotation.TableName
import java.math.BigDecimal

@TableName("tb_seed")
data class SeedEntity(
        @TableId
        var seedId: Long,
        var seedName: String?,
        var seedGrade: Int?,
        var seedProdgold: Int?,
        var seedProdred: BigDecimal?,
        var seedExperience: Int?,
        var seedGold: Int?,
        var seedImg: String?
)
